src/screens/home/components/HomeView.js

```diff
@@ -169,10 +172,12 @@ const HomeView = ({ route, navigation, setLauncherContext, setBarStyle }) => {
             href: cozyAppFallbackURL,
             slug
           })
+        } else {
+          if (shouldWaitCozyApp === undefined) setShouldWaitCozyApp(false)
```

Comment:
What is the purpose of setting shouldWaitCozyApp to undefined by default ?

Reply:
This is because a render can be done before the first `useEffect` execution. If defaulted to `false` the WebView would render without waiting for the `useEffect` to set the `shouldWaitCozyApp`.